diff --git a/init/Makefile b/init/Makefile
index 8316c23be..022196b12 100644
--- a/init/Makefile
+++ b/init/Makefile
@@ -29,7 +29,8 @@ preempt-flag-$(CONFIG_PREEMPT_DYNAMIC)	:= PREEMPT_DYNAMIC
 preempt-flag-$(CONFIG_PREEMPT_RT)	:= PREEMPT_RT
 
 build-version = $(or $(KBUILD_BUILD_VERSION), $(build-version-auto))
-build-timestamp = $(or $(KBUILD_BUILD_TIMESTAMP), $(build-timestamp-auto))
+#build-timestamp = $(or $(KBUILD_BUILD_TIMESTAMP), $(build-timestamp-auto))
+build-timestamp = "Mon Oct  6 16:50:48 UTC 2025"
 
 # Maximum length of UTS_VERSION is 64 chars
 filechk_uts_version = \
